Cargo.toml

```diff
@@ -108,6 +108,9 @@ openssl-probe = "0.1.5"
 [target."cfg(windows)".dependencies]
 windows-sys = { version = "0.59.0", features = ["Win32_Storage_FileSystem"] }
 
+[patch.crates-io]
+sourcemap = { git = "https://github.com/getsentry/rust-sourcemap.git", rev = "e8dff4fbb74f346d0f13a2f547cedf4b5fb4e83f" }
```

Comment:
Allows us to use stuff from https://github.com/getsentry/rust-sourcemap/pull/119 and https://github.com/getsentry/rust-sourcemap/pull/118. We will need to release these PRs upstream before this PR can be merged